packages/mobile/src/identity/phoneNumPrivacyService.ts

```diff
@@ -1,23 +1,26 @@
 // Utilities for interacting with the Phone Number Privacy Service service (aka PGPNP)
 
-import { ContractKit } from '@celo/contractkit'
+import { ec as EC } from 'elliptic'
 import { ErrorMessages } from 'src/app/ErrorMessages'
 import networkConfig from 'src/geth/networkConfig'
 import Logger from 'src/utils/Logger'
+const ec = new EC('secp256k1')
 
 const TAG = 'identity/phoneNumPrivacyService'
 
 export async function postToPhoneNumPrivacyService<ResponseType>(
```

Comment:
Are we planning on moving this too?

Reply:
Ah this was already moved (see `phone-number-lookup.ts`). Just forgot to remove it from here.. thanks!